DesktopUI2/DesktopUI2/ViewModels/StreamSelectorViewModel.cs

```diff
@@ -147,53 +146,51 @@ private async Task GetStreams()
         return;
 
       ShowProgress = true;
+      
+      //needed for the search feature
+      StreamGetCancelTokenSource?.Cancel();
+      StreamGetCancelTokenSource = new CancellationTokenSource();
 
-      try
-      {
-        //needed for the search feature
-        StreamGetCancelTokenSource?.Cancel();
-        StreamGetCancelTokenSource = new CancellationTokenSource();
+      var streams = new List<StreamAccountWrapper>();
 
-        var streams = new List<StreamAccountWrapper>();
+      foreach (var account in Accounts)
+      {
+        if (StreamGetCancelTokenSource.IsCancellationRequested)
+          return;
 
-        foreach (var account in Accounts)
+        try
         {
+          var result = new List<Stream>();
+
+          //NO SEARCH
+          if (SearchQuery == "")
+            result = await account.Client.StreamsGet(StreamGetCancelTokenSource.Token, 25);
+          //SEARCH
+          else
+            result = await account.Client.StreamSearch(StreamGetCancelTokenSource.Token, SearchQuery, 25);
+
           if (StreamGetCancelTokenSource.IsCancellationRequested)
             return;
 
-          try
-          {
-            var result = new List<Stream>();
-
-            //NO SEARCH
-            if (SearchQuery == "")
-              result = await account.Client.StreamsGet(StreamGetCancelTokenSource.Token, 25);
-            //SEARCH
-            else
-              result = await account.Client.StreamSearch(StreamGetCancelTokenSource.Token, SearchQuery, 25);
-
-            if (StreamGetCancelTokenSource.IsCancellationRequested)
-              return;
-
-            streams.AddRange(result.Select(x => new StreamAccountWrapper(x, account.Account)));
-
-          }
-          catch (Exception e)
-          {
-            if (e.InnerException is System.Threading.Tasks.TaskCanceledException)
-              return;
-            new SpeckleException("Could not fetch streams", e, true, Sentry.SentryLevel.Error);
-          }
+          streams.AddRange(result.Select(x => new StreamAccountWrapper(x, account.Account)));
         }
-        if (StreamGetCancelTokenSource.IsCancellationRequested)
-          return;
-
+        catch (Exception ex)
+        {
+          bool isError = ex is TaskCanceledException || ex.InnerException is TaskCanceledException;
```

Comment:
the logic here should be reversed no? this should be an `isNotConsideredAsError` variable as we do not care about task cancellations.

Reply:
Good catch!